pkg/operator/ceph/cluster/osd/health.go

```diff
@@ -94,6 +97,32 @@ func (m *OSDHealthMonitor) Update(removeOSDsIfOUTAndSafeToRemove bool) {
 
 // checkOSDHealth takes action when needed if the OSDs are not healthy
 func (m *OSDHealthMonitor) checkOSDHealth() error {
```

Comment:
If we're just logging the errors as suggested below, no need to return any error here
```suggestion
func (m *OSDHealthMonitor) checkOSDHealth() {
```

Reply:
fixed